cli/src/commands/backout.rs

```diff
@@ -47,12 +47,40 @@ pub(crate) fn cmd_backout(
         parents.push(destination);
     }
     let mut tx = workspace_command.start_transaction();
-    back_out_commit(
-        command.settings(),
-        tx.mut_repo(),
-        &commit_to_back_out,
-        &parents,
-    )?;
+    let commit_to_back_out_subject = commit_to_back_out
+        .description()
+        .lines()
+        .next()
+        .unwrap_or_default();
+    let is_reapplying_backed_out_commit = commit_to_back_out_subject.starts_with("Revert \"")
```

Comment:
I admit I quite like the term `backout`, though I totally agree that we should include the commit message of the backed out change in the title as Ollie originally wanted. Could we say `"Back out \"` here instead? This is minor but it seems more self-consistent, even if `git revert` uses a different verb.

Reply:
I don't have a strong opinion on this, plus it's easy enough to change the message description after if required, so I'll make the change. Should we still call the backing out of a backed out commit "Reapply" like `git revert`?